src/Files.App/Helpers/FileOperationsHelpers.cs

```diff
@@ -0,0 +1,909 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.IO.Pipes;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Common;
+using Files.App.DataModels;
+using Files.App.Filesystem.Permissions;
+using Files.App.Shell;
+using Files.Shared;
+using Files.Shared.Extensions;
+using Microsoft.Win32;
+using Tulpep.ActiveDirectoryObjectPicker;
+using Vanara.PInvoke;
+using Vanara.Windows.Shell;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Foundation.Collections;
+
+namespace Files.App.Helpers
+{
+    public class FileOperationsHelpers
+    {
+        private static FileTagsDb dbInstance = new(Path.Combine(Windows.Storage.ApplicationData.Current.LocalFolder.Path, "filetags.db"));
+        private static readonly JsonElement defaultJson = JsonSerializer.SerializeToElement("{}");
```

Comment:
Is this still necessary?

Reply:
For the DB instance yes, I'll work on removing the JSON default later